vocabs/v1/context.jsonld

```diff
@@ -1,89 +1,59 @@
 {
   "@context": {
-    "@version": 1.1,
+    "@protected": true,
     "id": "@id",
     "type": "@type",
-    "dc": "http://purl.org/dc/terms/",
-    "schema": "http://schema.org/",
-    "sec": "https://w3id.org/security#",
-    "didns": "https://www.w3.org/ns/did#",
-    "xsd": "http://www.w3.org/2001/XMLSchema#",
-    "EcdsaSecp256k1VerificationKey2019": "sec:EcdsaSecp256k1VerificationKey2019",
-    "Ed25519Signature2018": "sec:Ed25519Signature2018",
-    "Ed25519VerificationKey2018": "sec:Ed25519VerificationKey2018",
-    "JsonWebKey2020": "sec:JsonWebKey2020",
-    "JsonWebSignature2020": "sec:JsonWebSignature2020",
-    "Bls12381G1Key2020": "sec:Bls12381G1Key2020",
-    "Bls12381G2Key2020": "sec:Bls12381G2Key2020",
-    "RsaVerificationKey2018": "sec:RsaVerificationKey2018",
-    "SchnorrSecp256k1VerificationKey2019": "sec:SchnorrSecp256k1VerificationKey2019",
-    "X25519KeyAgreementKey2019": "sec:X25519KeyAgreementKey2019",
-    "ServiceEndpointProxyService": "didns:ServiceEndpointProxyService",
-    "LinkedDomains": "https://identity.foundation/.well-known/resources/did-configuration/#LinkedDomains",
+
     "alsoKnownAs": {
       "@id": "https://www.w3.org/ns/activitystreams#alsoKnownAs",
       "@type": "@id",
       "@container": "@set"
     },
     "assertionMethod": {
-      "@id": "sec:assertionMethod",
+      "@id": "https://w3id.org/security#assertionMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "authentication": {
-      "@id": "sec:authenticationMethod",
+      "@id": "https://w3id.org/security#authenticationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityDelegation": {
-      "@id": "sec:capabilityDelegationMethod",
+      "@id": "https://w3id.org/security#capabilityDelegationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "capabilityInvocation": {
-      "@id": "sec:capabilityInvocationMethod",
+      "@id": "https://w3id.org/security#capabilityInvocationMethod",
       "@type": "@id",
       "@container": "@set"
     },
     "controller": {
-      "@id": "sec:controller",
+      "@id": "https://w3id.org/security#controller",
       "@type": "@id"
     },
-    "created": {
-      "@id": "dc:created",
-      "@type": "xsd:dateTime"
-    },
-    "blockchainAccountId": "sec:blockchainAccountId",
     "keyAgreement": {
-      "@id": "sec:keyAgreementMethod",
+      "@id": "https://w3id.org/security#keyAgreementMethod",
       "@type": "@id",
       "@container": "@set"
     },
-    "publicKey": {
-      "@id": "sec:publicKey",
-      "@type": "@id",
-      "@container": "@set"
-    },
-    "publicKeyBase58": "sec:publicKeyBase58",
-    "publicKeyJwk": {
-      "@id": "sec:publicKeyJwk",
-      "@type": "@json"
-    },
     "service": {
-      "@id": "didns:service",
-      "@type": "@id",
-      "@container": "@set"
-    },
-    "serviceEndpoint": {
-      "@id": "didns:serviceEndpoint",
-      "@type": "@id"
-    },
-    "updated": {
```

Comment:
Do we want to leave this in here and add the other did document metadata object data in here as well since it's being defined in [DID Document Metadata](https://w3c.github.io/did-core/#did-document-metadata)

Reply:
I do not think so. The value MUST be the URL that identifies that property. A separate vocabulary specification may refer to the place where that property is defined in human terms. But that is not the role of a context file.

Reply:
Do we expect to have a separate context where the DID Document Metadata did-core properties will be defined then? I'm thinking about the case where the entire DID resolution object is JSON-LD and how we would go about making that work. This may make more sense to include that context in the resolution work as well.

Reply:
I think this should go into https://github.com/w3c-ccg/did-resolution/blob/gh-pages/contexts/did-resolution-v1.json.

Reply:
Agree with @peacekeeper. 

I'll also note that some people were cranky about DID Resolution results being JSON-LD, so I'd be careful about making that assumption. I defer to @peacekeeper on the latest there, though... it would be nice to have some light semantics that weren't just JSON for DID Resolution results... namely because we might be able to use that w/ `did:web` to sign entire DID Documents *and* encode other DID Resolution things in the `did:web` file on the server. Just some things to keep in mind...

Reply:
Cool, I'm aligned with keeping that to did-resolution spec. That resolve this for me.